llvm/lib/Support/README.md.SipHash

```diff
@@ -0,0 +1,126 @@
+# SipHash
+
+[![License:
+CC0-1.0](https://licensebuttons.net/l/zero/1.0/80x15.png)](http://creativecommons.org/publicdomain/zero/1.0/)
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
This README doesn't clearly indicate that the siphash code is also available under the Apache-2.0 WITH LLVM-exception license.
However, at the end of this README, it is stated clearly that the code is available under the Apache-2.0 WITH LLVM-exception license.
We also know from talking with the authors/copyright holders that they explicitly made this available under the Apache-2.0 WITH LLVM-exception license.

I also see that this README file is removed in the follow-on commit (#94394).

Therefore, I would not import this README file at all, to avoid any potential confusion triggered by this not-fully-correct statement at the start of the README file here.

Reply:
Yes, I added it only because the text at the end was the only mention of the license I had found.  Happy to remove it!